docs/aspnet4x-mvc.md

```diff
@@ -174,7 +174,11 @@ This example assumes you have a view called `Claim` whose model is of type `Syst
 
 ## Handling failures
 
-In the event a failure occurs, the Okta.AspNet library provides the `OnAuthenticationFailed` delegate defined on the `OktaMvcOptions` class. The following is an example of how to use `OnAuthenticationFailed` to handle authentication failures:
+library exposes [OpenIdConnectEvents](https://docs.microsoft.com/en-us/previous-versions/aspnet/mt180963(v=vs.113)) so you can hook into specific events during the authentication process. For more information see [`AuthenticationFailed`](https://docs.microsoft.com/en-us/previous-versions/aspnet/mt180967(v=vs.113)).
```

Comment:
Looks like the beginning of the sentence was removed.  Should it read `This library` or `The Okta.AspNet library`?

Reply:
yes, good catch 👍